theme/less/_navigation.less

```diff
@@ -1,26 +1,56 @@
-nav
+header nav
 {
-  ul
+  border-top: 1px solid #E9E9E9;
+}
+
+header nav ul
+{
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+header nav ul a
+{
+  display: block;
+  font-size: 24px;
+  text-decoration: none;
+  color: @primary;
+  padding: 25px;
+}
+
+@media (max-width: @tablet-width)
+{
+  header nav
   {
-    li
-    {
-      a
-      {
-        &:hover
-        {
-          text-decoration: underline;
-        }
-        text-decoration: none;
-        color: @reverse;
-      }
-      display: inline-block;
-      padding: 10px 10px 10px 0;
-    }
-    padding: 0 30px;
-    margin: 0;
-    list-style: none;
-    max-width: 1000px;
-    margin: 0px auto;
+    border-top: none;
+    text-align: center;
+    overflow: hidden;
+    transition: height .5s ease-in-out;
   }
-  background-color: @secondary;
-}
+
+  #hamburger
+  {
+    cursor: pointer;
+    border: 0;
+    width: 32px;
+    background: url(/images/hamburger.svg) transparent no-repeat center;
```

Comment:
Meeting notes: We agreed to move background to the ::after tag, similar to how is done in cmints.io.

Reply:
Done.